components/heading/examples/00-Heading.js

```diff
@@ -1,138 +1,141 @@
 /** @jsx jsx */
 
 import { GEL, jsx } from '@westpac/core';
+import { Fragment } from 'react';
 import { Heading } from '@westpac/heading';
 
 function Example({ brand }) {
 	return (
-		<GEL brand={brand}>
-			<h2>
-				Without <code>tag</code> prop
-			</h2>
-			<Heading size={1}>Heading size: 1</Heading>
-			<Heading size={2}>Heading size: 2</Heading>
-			<Heading size={3}>Heading size: 3</Heading>
-			<Heading size={4}>Heading size: 4</Heading>
-			<Heading size={5}>Heading size: 5</Heading>
-			<Heading size={6}>Heading size: 6</Heading>
-			<Heading size={7}>Heading size: 7</Heading>
-			<Heading size={8}>Heading size: 8</Heading>
-			<Heading size={9}>Heading size: 9</Heading>
-			<Heading size={10}>Heading size: 10</Heading>
+		<Fragment>
```

Comment:
It's a long diff but I can't see why there is a `Fragment` used...?

Reply:
Was left over from something I was playing with. Now removed.